Gemfile

```diff
@@ -13,6 +13,7 @@ gem 'sidekiq', require: false
 gem 'aws-sdk-sqs', require: false
 gem 'shoryuken', require: false
 
+gem 'kaminari', require: false
```

Comment:
Why do we need this?

Reply:
I think, we do not need this file. 

Before I thought Gemfile should has all the gems from old Appraisals file. But now I see the way  Chewy works with gems.